src/lib/setup.ts

```diff
@@ -7,6 +7,9 @@ import {
 
 export const baseUrl = getEnvironmentVariable("BASE_URL", "http://localhost:3001");
 
+export const pushApiVersion = getEnvironmentVariable("PUSH_API_VERSION", "1.0");
+export const pullApiVersion = getEnvironmentVariable("PULL_API_VERSION", "1.0");
```

Comment:
Only suggestion: I would remove the default value. Let's make it explicit in env var.
```suggestion
export const pushApiVersion = getEnvironmentVariable("PUSH_API_VERSION");
export const pullApiVersion = getEnvironmentVariable("PULL_API_VERSION");
```